ProjectSidewalk/SidewalkWebpage
# 1464 insufficient landing space tag

Resolves #1464 

Changes:
* Adds a "not enough landing space" tag to the curb ramp context menu.
* Adds a section to the tutorial for adding a "not enough landing space" tag to the 2nd label.

Testing instructions:
1. Open the audit page and place a curb ramp label. Try the new landing space tag (including with a keyboard shortcut) and make sure that the tag is added/removed from the `label_tag` table appropriately.
1. Go through the tutorial and make sure that the new section for the tag on the 2nd label works as intended and is worded well.
1. Go through the tutorial and carefully make sure that the mission completion percentage goes up an appropriate amount each time (please be careful to test all cases!).

Old:
![curb-ramp-context-menu-old](https://user-images.githubusercontent.com/6518824/53924771-35671000-4032-11e9-8db0-eb0b698bc357.png)
New:
![curb-ramp-context-menu-after](https://user-images.githubusercontent.com/6518824/53924774-37c96a00-4032-11e9-84b1-83e84bd58a0f.png)

Wording for the new tutorial section:
![landing-space-tutorial-1](https://user-images.githubusercontent.com/6518824/53924943-f9807a80-4032-11e9-9c79-e7688d599900.png)
![landing-space-tutorial-2](https://user-images.githubusercontent.com/6518824/53924945-fab1a780-4032-11e9-89d9-ff8f664ac9b6.png)
